images/context.puml

```diff
@@ -1,21 +1,28 @@
-@startuml
+@startuml High_level
 !include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Context.puml
 
-Person(user_data_input, "Data input")
-Person(user_data_request, "Data discover and request")
-Person(user_admin, "Admin", "Data contrdollers or data managers")
+Person(user_input, "Data input")
+Person(user_request, "Data discover and request")
+Person(user_admin, "Admin", "Data controllers or data managers")
 
 Person_Ext(user_results, "Explore findings and projects")
 
-Lay_Right(user_admin, user_results)
+Lay_R(user_admin, user_input)
+Lay_R(user_input, user_results)
+Lay_R(user_results, user_request)
+
+
+AddRelTag("line", $lineStyle = DashedLine())
 
 System(ui, "Data Resource structured by Seedcase")
 System_Ext(external_server, "External secure server")
 
-Rel(user_data_input, ui, "Uploads data")
-Rel(user_data_request, ui, "Find and request data for project")
-Rel(user_results, ui, "Explore findings of study")
+Rel(user_input, ui, "Uploads data")
+Rel(user_request, ui, "Find and request data for project")
 Rel(user_admin, ui, "Manages issues and requests for data projects")
+Rel(user_results, ui, "Explore findings of study")
```

Comment:
```suggestion
Rel(user_results, ui, "Explore variables and projects")
```
Wanted to add that the shopping card will include that they can explore what variables are within the data resource - but not make it sound like everyone can explore the actual data. Not sure that I succeeded.